fission-core/library/Fission/Text/Encoded/Class.hs

```diff
@@ -0,0 +1,29 @@
+module Fission.Text.Encoded.Class (ToEncoded (..)) where
+
+import           RIO
+import qualified RIO.ByteString.Lazy as Lazy
+
+import           Fission.Text.Encoded.Types
+import           Fission.Text.Encoding.Types
+
+-- | A helper for beinging some native text into the Encoded type and tagging
```

Comment:
Typo "beinging/bringing" but :man_shrugging:

Reply:
hahhahaa and I'm the native English speaker! I fat finger a _lot_ of typos these days. Thanks for catching that!